### PR TITLE
Static Pages

### PR DESCRIPTION
**Pull request by tranxuanhaibk**
## Related Tickets
- [#43417](https://edu-redmine.sun-asterisk.vn/projects/oe50b_ssf/issues)

## WHAT (optional)
- Tạo các page cơ bản, thiết lập i18n

## HOW
- Cài gem boootstrap, gem i18n

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
Screenshot from 2021-11-19 15-14-45
Screenshot from 2021-11-19 15-17-43

### CODE REVIEW

**Comment by nguyenvanson332000** [comment]
ready

**Comment by tranxuanhaibk** [comment]
ready

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (quoctd-0607) [inline app/helpers/static_pages_helper.rb]: remove file này đi

Comment (tranxuanhaibk) [reply app/helpers/static_pages_helper.rb]: dạ vâng ạ

**Comment by tranxuanhaibk** [comment]
ready

**Pull request merged**